she_codes_news/news/models.py

```diff
@@ -1,8 +1,26 @@
+from django.contrib.auth import get_user_model
 from django.db import models
 
+USER = get_user_model()
+
 
 class NewsStory(models.Model):
     title = models.CharField(max_length=200)
-    author = models.CharField(max_length=200)
+   # author = models.CharField(max_length=200)
+    author = models.ForeignKey(
+        USER, on_delete=models.CASCADE, related_name="stories"
+    )
     pub_date = models.DateTimeField()
     content = models.TextField()
+    class Meta:
+        ordering = ["-pub_date"]
+    
+    # Add an image url
+    image_url = models.URLField(max_length=200, null= True)
```

Comment:
typically your fields should go above the class Meta, this is just to make it more readable